gestionatr/input/messages/B70.py

```diff
@@ -610,8 +616,8 @@ def get_periode_factura(self):
             - data fi: la mes nova de les fechasta dels conceptes
         """
         return (
-            min([x.fecdesde for x in self.listaconceptos]),
-            max([x.fechasta for x in self.listaconceptos])
+            min([x.fecdesde for x in self.listaconceptos if x.codconcepto not in self.productes_exclosos]),
+            max([x.fechasta for x in self.listaconceptos if x.codconcepto not in self.productes_exclosos])
```

Comment:
El valor per defecte de "productes_exclosos" es `None`. Aixo vol dir que si algu instancia un B70 sense passar productes_exclosos aquesta linia petaria ja que no es pot fer un `in None`. Fes un IF o inicialitza a llista buida